app/templates/road-sign-concepts/road-sign-concept.hbs

```diff
@@ -12,26 +12,36 @@
       <c.header @badgeSkin="brand">
         <div class="flex-content-between">
           <div class="no-flex-shrink">
-            <img class="au-u-margin-right" alt={{t 'abb.name'}} src={{@model.image}}>
+            <img class="au-u-margin-right" alt={{t 'abb.name' }} src={{@model.image}}>
           </div>
           <div>
             <AuHeading @level="2" @skin="4">
               {{ @model.roadSignConceptCode}} -
               {{#each @model.categories as |category|}}
-                <span>{{ category.label}}</span>
+              <span>{{ category.label}}</span>
```

Comment:
I think the indentation was fine before?

Reply:
indeed, can be reverted